.gitignore

```diff
@@ -1,3 +1,52 @@
+# Byte-compiled / optimized / DLL files
 __pycache__/
-*.idea
-*.DS_store
+*.py[cod]
+*$py.class
+
+# Mac/OSX
+.DS_Store
+
+# Windows
+Thumbs.db
+
+# PyCharm
+.idea/
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# Unit test / coverage reports
+.coverage
+.coverage.*
+.cache
+.pytest_cache/
+
+# Sphinx documentation
+docs/_build/
```

Comment:
This can be removed. We don't use sphinx.

Reply:
Will remove it, I figured that sphinx may be useful for future documentation (e.g., for readthedocs) so I didn't remove it yet. In any case, it can be re-added if we ever decide to use it.